content/docs/2.4/operate/cluster.md

```diff
@@ -39,7 +39,7 @@ Here is an overview of all KEDA deployments and the supported replicas:
 | Deployment     | Support Replicas | Reasoning                                                                                                        |
 | -------------- | ---------------- | ---------------------------------------------------------------------------------------------------------------- |
 | Metrics Server | 1                | Limitation in [k8s custom metrics server](https://github.com/kubernetes-sigs/custom-metrics-apiserver/issues/70) |
-| Operator       | 1                |                                                                                                                  |
+| Operator       | 2                | While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic. You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce a downtime during a failover.|
```

Comment:
```suggestion
| Operator       | 2                | While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic. You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.<br /><br />This is only supported as of KEDA v2.6 if you are using our Helm chart.|
```